crates/nu-command/src/filesystem/ls.rs

```diff
@@ -131,9 +131,13 @@ impl Command for Ls {
                         Err(e) => {
                             if e.kind() == ErrorKind::PermissionDenied
                                 || e.kind() == ErrorKind::Other
+                            // on windows this is "The process cannot access the file
+                            // because it is being used by another process. (os error 32)"
+                            || e.raw_os_error() == Some(32)
                             {
                                 None
                             } else {
+                                eprintln!("raw {:?}", e.raw_os_error());
```

Comment:
we could make this part of the error we show the user rather than a separate println

Reply:
oops - didn't mean to leave that in